internal/scheduler/jobs.go

```diff
@@ -48,3 +51,30 @@ func (j *CheckUpdatesJob) Run() {
 		j.lastCheckVersion = newRelease.TagName
 	}
 }
+
+type TempDirCleanupJob struct {
+	Name string
+	Log  zerolog.Logger
+}
+
+func (j *TempDirCleanupJob) Run() {
+	j.Log.Debug().Msg("Starting cleanup of temporary directory.")
+
+	tmpFilePattern := "autobrr-"
+	tmpDir := os.TempDir()
+
+	files, err := os.ReadDir(tmpDir)
+	if err == nil {
+		for _, file := range files {
+			if strings.HasPrefix(file.Name(), tmpFilePattern) {
+				tempFiles := filepath.Join(tmpDir, file.Name())
+				fileInfo, err := os.Stat(tempFiles)
+				if err == nil {
+					if fileInfo.ModTime().Before(time.Now().Add(-time.Hour)) {
+						os.Remove(tempFiles)
+					}
+				}
+			}
+		}
+	}
```

Comment:
I think it's wise to add error handling to both os.Readdir, os.Stat and os.Remove functions and log errors.

Do we possibly have to check for the owner of the files in case of shared systems?

And maybe have to age check only delete files older than 24h? In case someone is doing backup or something else with the files.

Could also add a counter and log the amount of deleted files at the end. If you want to make it even fancier sum the size of the files as well and log that on the same line.